src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -463,6 +464,16 @@
         {:status 200
          :body (get-code-content environment code-id file-path)}))))
 
+(schema/defn http-client-metrics-summary
+  :- {:metrics-data [http-client-common/MetricIdMetricData]
+      :sorted-metrics-data [http-client-common/MetricIdMetricData]}
+  [metric-registry :- MetricRegistry
+   metric-ids-to-select :- (schema/atom [[schema/Str]])]
```

Comment:
Since we'd reference this in a few places, might be nice to factor the `(schema/atom [[schema/Str]])` out into a separate def.